go/plek.go

```diff
@@ -37,12 +37,9 @@ var httpDomains = map[string]bool{
 }
 
 // Find returns the base URL for the given service name in the default parent
-// domain. The domain is taken from the GOVUK_APP_DOMAIN environment variable.
-// If this is unset, "dev.gov.uk" is used.
```

Comment:
We should keep this detail about the `GOVUK_APP_DOMAIN` env var here.  Only exported types/functions form part of the [public godoc](http://godoc.org/github.com/alphagov/plek/go).  My moving this comment to the private function below, it effectively removes it from the public documentation.